dashboard/modules/snapshot/snapshot_schema.json

```diff
@@ -192,6 +207,8 @@
             "type": "string"
           },
           "required": [
+            "rayVersion",
```

Comment:
this is good

Reply:
do we need to merge master? I thought this is already merged?

Reply:
It was, I just forgot to put it in the "required" field in the last PR.